geonode_mapstore_client/client/js/plugins/MetadataEditor/components/_fields/SchemaField.jsx

```diff
@@ -37,6 +41,9 @@ const SchemaField = (props) => {
     const isSingleSelect = schema?.type === 'object' && !isEmpty(schema?.properties);
 
     if (autocomplete && (isMultiSelect || isSingleSelect)) {
+        const errors = !hideError ? isArray(errorSchema)
+            ? errorSchema.map(error => get(error, 'id.__errors', [])).flat()
+            : get(errorSchema, '__errors', []) : [];
```

Comment:
We should print all the possible message, here is focusing on the `id` key but we should take into account other possible property keys. Here a proposal, please review and fix if needed:

```js
const errors = (!hideError ? castArray(errorSchema) : [])
    .map((errorEntry) => {
        if (errorEntry?.__errors) {
            return {
                messages: errorEntry.__errors.map((message) => message)
            };
        }
        return errorEntry ? Object.keys(errorEntry).map((key) => {
            if (errorEntry[key]?.__errors) {
                return {
                    key,
                    messages: errorEntry[key].__errors.map((message) => message)
                };
            }
            return null;
        }) : null;
    }).filter(value => value !== null).flat();
```

Reply:
Yeah looking at the schema example of errors in the other issue it makes sense. I will update it accordingly. Thanks!